sample_size/scripts/sample_size_run.py

```diff
@@ -14,20 +14,23 @@ def main() -> None:
     """
     from sample_size.sample_size_calculator import SampleSizeCalculator
     from sample_size.scripts.input_utils import get_alpha
-    from sample_size.scripts.input_utils import get_metric_metadata
+    from sample_size.scripts.input_utils import get_metrics
+    from sample_size.scripts.input_utils import get_variants
 
     try:
         # Get alpha for power analysis
         alpha = get_alpha()
-        calculator = SampleSizeCalculator(alpha)
+        variants = get_variants()
+        calculator = SampleSizeCalculator(alpha, variants)
 
-        # register metric
-        metric_type, metric_metadata = get_metric_metadata()
-        calculator.register_metric(metric_type, metric_metadata)
+        metrics = get_metrics()
```

Comment:
Do we need to update Readme file now that we will support multiple metrics?

Reply:
Yes I'm planning on doing that in my last PR(there's one more PR coming for the code change)

Reply:
sounds good